_stylesheets/docs.css

```diff
@@ -927,7 +926,7 @@ h2:hover>a.anchor, h2>a.anchor:hover, h3:hover>a.anchor, h3>a.anchor:hover, h4:h
 
   .main{
     width: 100%;
-    margin-top: 165px;
+    margin-top: 200px;
```

Comment:
**nitpick:** Maybe `175px`, too much whitespace before the title on Mobile screens.

LGTM even without fixing this nitpick.

Reply:
Actually put this back to 165px. Applied in preview.